Watts.nuspec

```diff
@@ -2,15 +2,15 @@
 <package>
   <metadata>
     <id>WebApiToTypeScript</id>
-    <version>1.11.1</version>
+    <version>2.0.0</version>
     <authors>Balakrishnan (Balki) Ranganathan</authors>
     <owners>Balakrishnan (Balki) Ranganathan</owners>
     <licenseUrl>https://raw.githubusercontent.com/greymind/WebApiToTypeScript/master/LICENSE</licenseUrl>
     <projectUrl>https://github.com/greymind/WebApiToTypeScript</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>A tool for code generating TypeScript endpoints, interfaces, enums, resources and view paths for your ASP.NET Web API controllers/web apps</description>
     <summary>A tool for code generating TypeScript endpoints, interfaces, enums, resources and view paths for your ASP.NET Web API controllers/web apps</summary>
-    <releaseNotes>Fix getting base types in certain cases</releaseNotes>
+    <releaseNotes>Optimize code for execution and size when compiling TypeScript to ES5.</releaseNotes>
```

Comment:
Oh just noticed you added a config variable. In any case, can you please add what the breaking changes we need to look out for? We should also start a section on readme